mobile/src/components/shared/ICardPage.jsx

```diff
@@ -33,13 +36,40 @@ const MyICardPage = (props) => {
   }
 
 
+
+  //creation of two experimental buttons
+  const Button = ({status, onPress}) => {
+    let content1
+    if (status == 'iNactive') {
+      content1 = (
+        <View>
+          <TouchableOpacity style={styles.button} onPress={onPress}>
+         <Text style={styles.buttonText}> Verify account</Text>
+        </TouchableOpacity>
+        </View>
+      )
+    }
+    else if (status == 'Inactive') {
+      content1 = (
+        <View>
+          <TouchableOpacity style={styles.button} onPress={onPress}>
+         <Text style={styles.buttonText}> Reverify account</Text>
+        </TouchableOpacity>
+        </View>
+      )
+    }
+
+    return <Text>{content1} </Text>
+  }
+
```

Comment:
Since the code is mostly duplicated, you can merge this into one view function and just use 
`a ? 'Verify account' : b 'Reverify account'` in the text field to conditionally render it.

Reply:
Also good to name it something like VerificationButton instead of just Button.